bot/bot.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import os
+
+import disnake
+from disnake.ext import commands
+
+logger = logging.getLogger(__name__)
+
+try:
+    import dotenv
+    dotenv.load_dotenv()
+except ModuleNotFoundError:
+    pass
+
+DEV_LOG_CHANNEL = os.environ.get("DEV_LOG_CHANNEL")
+TOKEN = os.environ.get("TOKEN")
```

Comment:
I believe this should be moved to a `constants.py` file to be consistent with other projects, and for a place to add more constants.